File: Spinner2/app/src/main/java/com/busrayasar/spinner/MainActivity.java
package com.busrayasar.spinner;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.AdapterView;
import android.widget.ArrayAdapter;
import android.widget.Spinner;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {


    Spinner spinner;
    String [] istanbulIlceleri = {"Çekmeköy","Ümraniye","Üsküdar","Kadıköy","Fatih","Başakşehir","Beşiktaş", "Şişli" };
    ArrayAdapter arrayAdapter;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        spinnerTanimla();
        adapterOlustur();
        spinneraAdapterEkle();
        bilgiVer();
    }
    public void spinnerTanimla(){
        spinner = findViewById(R.id.spinner);
    }
    public void adapterOlustur(){
        arrayAdapter=new ArrayAdapter(this,R.layout.support_simple_spinner_dropdown_item, istanbulIlceleri);
    }
    public void spinneraAdapterEkle(){
        spinner.setAdapter(arrayAdapter);
    }
    //tıklanan itema göre işlem yapma
    public void bilgiVer()
    {
        spinner.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
            @Override
            public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
                Toast.makeText(getBaseContext() , "" + spinner.getSelectedItem().toString(), Toast.LENGTH_LONG).show();
            }

            @Override
            public void onNothingSelected(AdapterView<?> parent) {

            }
        });
    }
}

File: SuperKahraman/app/src/main/java/com/busrayasar/superkahraman/MainActivity.kt
package com.busrayasar.superkahraman

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    var num = 0
    var name = ""
    var age : Int? = null
    var job = ""
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
            //Scope - kapsam
        num = 10
    }

    fun makeSuperHero(view: View){
        num = 15
        // finding the textView
         name =  nameText.text.toString()
         age = ageText.text.toString().toIntOrNull()
         job = jobText.text.toString()

        if (age == null){
            textsuperHero.text = "Enter a valid age : "
        }else {
            val superhero = SuperHero(name, age!!, job)
            textsuperHero.text = "Name: ${superhero.name} Age: ${superhero.age} Job: ${superhero.job}"
        }

    }
}